crates/rspack_plugin_javascript/src/dependency/esm/esm_import_specifier_dependency.rs

```diff
@@ -349,11 +349,33 @@ impl DependencyTemplate for ESMImportSpecifierDependencyTemplate {
         *runtime,
         &compilation.module_graph_cache_artifact,
       )
-      && !module_graph
-        .get_exports_info(con.module_identifier())
-        .get_used_name(&module_graph, *runtime, ids)
-        .map(|used| used.is_inlined())
-        .unwrap_or_default()
+      && {
```

Comment:
I can see there are many similar pattern like this, a few lines of code refactored to many lines of code, can we have a facade function for this pattern, to make more cleaner at the call site, and more easier to use